### PR TITLE
logged in user cart functionality

### PR DESCRIPTION
**Pull request by irais-valenzuela**
# Context 
Needed to have a persistent cart for logged in users with CRUD abilities for their carts

# What this PR does
Displays a persistent cart for logged in user's even when they log out and come back in. Allows logged in users to perform CRUD operations on their carts. 

# How to test
Run server, open browser, login first user in database, and perform CRUD operations on cart. Log in and out to see if cart items persist.

### CODE REVIEW

**Comment by irais-valenzuela** [comment]
Let me know when someone reviews my PR so I can resolve the merge conflicts bcs I might need some code snippets.

File: client/Routes.js [hunk 1]
```diff
@@ -25,6 +26,11 @@ class Routes extends Component {
         {isLoggedIn ? (
           <Switch>
             <Route path="/home" component={Home} />
+            <Route exact path="/products" component={AllProducts} />
+            <Route path="/products/:productId" component={SingleProduct} />
+            <Route path="/confirmation" component={Confirmation} />
+            {/* added if user logged in go to logged in cart -irais  */}
+            <Route path="/cart/loggedIn" component={LoggedInUserCart} />
```
Comment (MichelleMullane) [inline client/Routes.js]: I think we need to make this "exact path" so we don't get double-rendering of the guest & logged-in user cart.  Or maybe we need to add that to the guest cart route path?

File: client/components/Cart.js [hunk 2]
```diff
@@ -14,9 +14,13 @@ class SingleOrder extends React.Component {
         <div id="single-product-detail" className="row">
           <div className="column rm1" key={order.id}>
             <h1> Shopping Cart</h1>
-            {order.orderId ? ( <div><p>Order Number: {order.orderId}</p>
-            <p>Price: $ {order.price}</p>
-            <p>Quantity: {order.qty}</p></div>): null }
+            {order.orderId ? (
```
Comment (MichelleMullane) [inline client/components/Cart.js]: Question about this file.  The guest cart component is called "SingleOrder.js".  I see there's also a "LoggedInUserCart.js".  What is the "Cart.js" file for?

File: client/components/SingleProduct.js [hunk 3]
```diff
@@ -21,9 +25,16 @@ class SingleProduct extends React.Component {
             <h1>{product.name}</h1>
             <p>Price: $ {product.price}</p>
             <p>Description: {product.desc}</p>
-            <button type="button" onClick={()=> {
-              this.props.createSingleOrder(product)
-            }}>Add to Cart</button>
+            <button
```
Comment (MichelleMullane) [inline client/components/SingleProduct.js:26]: I had a lot of changes to this button as well for the guest cart.  We need to be careful of what we keep when merging.

File: public/index.html [hunk 4]
```diff
@@ -3,11 +3,16 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width initial-scale=1.0" />
-    <title>FS-App-Template</title>
+    <title>Green Garden</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
Comment (MichelleMullane) [inline public/index.html:8]: Love that you added these fonts 😍

File: public/style.css [hunk 5]
```diff
@@ -12,6 +11,10 @@ label {
   display: block;
 }
 
+nav {
+  font-family: var(--sans);
```
Comment (MichelleMullane) [inline public/style.css:15]: Nice use of css variables!

File: server/auth/index.js [hunk 6]
```diff
@@ -6,6 +6,7 @@ module.exports = router;
 
 router.post("/login", async (req, res, next) => {
   try {
+    console.log("req.body", req.body)
```
Comment (MichelleMullane) [inline server/auth/index.js:9]: Maybe delete this console log?

**Review by MichelleMullane — COMMENTED** [review]
Amazing work, Irais!  I just have a couple of questions before I do the merging.  I LOVE your explicit variable names and your comments are very helpful.

**Review by MichelleMullane — APPROVED** [review]

File: client/Routes.js (hunk 1 above)
Comment (irais-valenzuela) [reply client/Routes.js]: Good point! I'll do it for logged in cart

File: client/components/Cart.js (hunk 2 above)
Comment (irais-valenzuela) [reply client/components/Cart.js]: I believe Cart.js was the old name for SingleOrder.js. I remember you and Nadia changed the name. I have the old file.

File: server/auth/index.js (hunk 6 above)
Comment (irais-valenzuela) [reply server/auth/index.js:9]: Yes! I will.

**Review by MichelleMullane — APPROVED** [review]

**Pull request merged**